SudokuBF/SudokuBF.cpp

```diff
@@ -235,8 +235,15 @@ int bruteForce(std::string_view strField)
 
 int main(int argc, char *argv[])
 {
-	if (argc != 2)
+	if (argc != 2) {
```

Comment:
if (argc != 2)
{
Indentation used in this file/project:
{ from new line after if